loss-detection-app/docker-compose.vap.yml

```diff
@@ -17,7 +17,8 @@ version: '3.7'
 
 networks:
   loss-detection-app_edgex-network:
-    external: true
```

Comment:
The VAP compose isn't run when simulating, so I think the network should be defined in `loss-detection` compose like it was.

Reply:
Okay, addressed!